Assets/Scripts/InteractableController.cs

```diff
@@ -4,7 +4,12 @@
 
 public class InteractableController : MonoBehaviour
 {
-    public string text;
+    int textnumber = 0;
```

Comment:
This can be `private` too.

Reply:
and camelCase! `textNumber`

Reply:
This all makes sense and is good to know.